workflows/wdl/iss_published/recipe.py

```diff
@@ -0,0 +1,66 @@
+import starfish
+from starfish import FieldOfView
+from starfish.image import Filter
+from starfish.image import ApplyTransform, LearnTransform
+from starfish.spots import DetectSpots
+from starfish.types import Axes
+
+
+def process_fov(field_num: int, experiment_str: str):
+    """Process a single field of view of ISS data
+    Parameters
+    ----------
+    field_num : int
+        the field of view to process
+    experiment_str : int
+        path of experiment json file
+
+    Returns
+    -------
+    DecodedSpots :
+        tabular object containing the locations of detected spots.
+    """
+
+    fov_str: str = f"fov_{int(field_num):03d}"
+
+    # load experiment
+    experiment = starfish.Experiment.from_json(experiment_str)
+
+    print(f"loading fov: {fov_str}")
+    fov = experiment[fov_str]
+
+    # note the structure of the 5D tensor containing the raw imaging data
+    imgs = fov.get_image(FieldOfView.PRIMARY_IMAGES)
+    dots = fov.get_image("dots")
+    nuclei = fov.get_image("nuclei")
+
+    masking_radius = 15
+    print("Filter WhiteTophat")
+    filt = Filter.WhiteTophat(masking_radius, is_volume=False)
+
+    filtered_imgs = filt.run(imgs, verbose=True, in_place=False)
```

Comment:
Consider moving the registration to before filtering so you can do inplace processing.